appstore_tools/console.py

```diff
@@ -514,3 +514,7 @@ def run():
         sys.exit(error)
     except KeyboardInterrupt as error:
         sys.exit(error)
+
+
+if __name__ == '__main__':
```

Comment:
I know this is common for python modules, but is this needed when developing with poetry where the console command is already made available?  I don't have strong feelings either way, but I never needed it.

Reply:
it was easier for me since I don't use poetry :) I'll have a look at what it does and remove this

Reply:
should be good now!